exercises/armstrong-numbers/example.js

```diff
@@ -0,0 +1,11 @@
+const validate = input => {
+  const digits = (input + '').split('');
```

Comment:
You want to convert `input` from a `Number` to a `String`, right?  It would be nice to have this be more semantic.  We would also prefer the use of the spread operator since this is an ES6-focused track.  That sounds a little contradictory, eh?  But that's the style we would prefer on this track:

```javascript
const digits = [...String(input)];
```

Reply:
That's much cleaner. Thanks for the suggestion 😄 
Commit amended